### PR TITLE
Only show history tabs for certain domains

### LINKED ISSUE #6844: More info "Controls" tab naming
## Checklist

- [x] I have updated to the latest available Home Assistant version.
- [x] I have cleared the cache of my browser.
- [x] I have tried a different browser to see if it is related to my browser.

## The problem

The first tab on more info dialogs is now called "Controls" which fits well for anything that has interactive controls there - like Lights, Media Players, Remotes, Alarms, Thermostats, Counters, etc... 

But on some others like Sensors, Sun, Device Trackers, Weather, etc which are read only, this naming feels a bit wrong and weird? And for newer users it might be confusing if they're expecting to see controls when that's not actually supported.

image

## Expected behavior

I think for domains that offer interactivity it should display "Controls" and for others that are read-only it should display "Details" or "Info" or "State" or something else....

## Environment

- Home Assistant release with the issue:
- Last working Home Assistant release (if known):
- Browser and browser version:
- Operating system:

## State of relevant entities

```yaml

```

## Problem-relevant configuration

```yaml

```

## Javascript errors shown in your browser console/inspector

```txt

```

### LINKED ISSUE #6856: Dont show history tab in more info for read-only entities
## The request

Sensor and binary sensor entities don't have controls, so we should just show the history on the main/single tab.

### PR DESCRIPTION
**Pull request by bramkragten**
## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Review by balloob — APPROVED** [review]

File: src/dialogs/more-info/ha-more-info-dialog.ts [hunk 1]
```diff
@@ -164,6 +175,13 @@ export class MoreInfoDialog extends LitElement {
                     .stateObj=${stateObj}
                     .hass=${this.hass}
                   ></more-info-content>
+                  ${!CONTROL_DOMAINS.includes(domain) ||
+                  !this._computeShowHistoryComponent(entityId)
+                    ? ""
+                    : html` <ha-more-info-history
```
Comment (zsarnett) [inline src/dialogs/more-info/ha-more-info-dialog.ts:181]:
```suggestion
                    : html`<ha-more-info-history
```

File: src/dialogs/more-info/ha-more-info-dialog.ts [hunk 2]
```diff
@@ -127,15 +137,16 @@ export class MoreInfoDialog extends LitElement {
...
+          this._computeShowHistoryComponent(entityId)
             ? html`
                 <mwc-tab-bar
                   .activeIndex=${this._currTabIndex}
                   @MDCTabBar:activated=${this._handleTabChanged}
                 >
                   <mwc-tab
                     .label=${this.hass.localize(
-                      "ui.dialogs.more_info_control.controls"
+                      "ui.dialogs.more_info_control.details"
```
Comment (zsarnett) [inline src/dialogs/more-info/ha-more-info-dialog.ts:149]: Its not just details though... But I guess this works :)

Comment (zsarnett) [reply src/dialogs/more-info/ha-more-info-dialog.ts:149]: This is fine until we change the names of entities 😉 then I think we should change it to match what they are.

Comment (zsarnett) [reply src/dialogs/more-info/ha-more-info-dialog.ts:149]: Or we could make it Entity for now until we change the naming

**Review by zsarnett — APPROVED** [review]
Naming things sucks :)

**Pull request merged**